Ladybird/BrowserWindow.cpp

```diff
@@ -491,10 +491,8 @@ Tab& BrowserWindow::new_tab(QString const& url, Web::HTML::ActivateTab activate_
 
     // We *don't* load the initial page if we are connected to a WebDriver, as the Set URL command may come in very
     // quickly, and become replaced by this load.
-    if (m_webdriver_content_ipc_path.is_empty()) {
-        // We make it HistoryNavigation so that the initial page doesn't get added to the history.
-        tab_ptr->navigate(url, Tab::LoadType::HistoryNavigation);
-    }
+    if (m_webdriver_content_ipc_path.is_empty())
```

Comment:
Given the origin of this code in https://github.com/SerenityOS/serenity/commit/194ddca24f600b6dc080c41fc19b2d42bcb86239 and https://github.com/SerenityOS/serenity/commit/731fec525e901f9efad63d3d920144cc47485194# by @krkk I'm not sure that this is entirely correct.

The first commit suggests that we don't want the "orginal" `about:blank` page from an initial page load here. But, the second commit suggests that we're doing this for the initial page of new tabs as well, which is often not going to be `about:blank`. Adding logic here to check whether url is `about:blank` feels kind of awkward, but might be the right path forward?

Reply:
I also think that checking for ``about:blank`` is not the best option here. What do you think about adding it to the history without checking anything?
I just tested it and Chrome also keeps ``about:blank`` in the history, which is, in my opinion, the correct way because it is technically part of the browsing history.

Reply:
To be honest, I'm not sure. It affects WebDriver, and possibly WPT. @trflynn89 @kalenikaliaksandr any thoughts on how to resolve this initial page in history question?

Reply:
This WebDriver workaround was removed in 93ffa77370d34ea72a9d0dee1d3c172762ac1e66, and we no longer force a load of `about:blank` as of 6c7422e8b7c6af376ec4eac068ed6c6e5615d740, so these shouldn't be an issue now. Those commits caused a conflict here though so that'll need to be resolved.